source/sixve.f90

```diff
@@ -21,25 +23,23 @@ subroutine sumpos
   integer nSplit, ioStat, lineNo, i, j
   logical spErr, fErr
 
-  save
-
-  rewind 10
+  rewind(unit10)
   lineNo = 0
   do i=1,1000
```

Comment:
Magic number 1000; I guess the significance of that number is "only show the first 1000 pairs or the user will get blasted with output"? Is this documented anywhere (e.g. in a comment)?

Reply:
Your guess is as good as mine.

Reply:
Hmm, since it refers to 1000 particles it probably isn't an ancient feature...